Automattic/wp-calypso
# Style selection: Hide legacy designs that are variations of another design

#### Proposed Changes

This PR hides "legacy" designs that are variations of another design, when the `signup/design-picker-style-selection` flag is enabled. This is because such designs are achievable by selecting the correct style variation from the "parent" design.

Currently, such designs are only Twenty Twenty Two "variation" designs.

#### Testing Instructions

* Go to http://calypso.localhost:3000/setup/designSetup?siteSlug=<your site slug>
* Advance until the design picker step
* Verify that you don't see Twenty Twenty Two variations.

#### Pre-merge Checklist






Related to #67137
